aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1355,11 +1339,39 @@ impl AptosVM {
     pub fn simulate_signed_transaction(
```

Comment:
let's comment on the new behavior, i.e., first try to simulate with aggregators enabled, then without.
I think this is great to incorporate, and can be extended to provide the flag from caller if needed in the future.